NitroxPatcher/Patches/Dynamic/GoTo_Patch.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Reflection;
+using HarmonyLib;
+using NitroxModel.Helper;
+using UnityEngine;
+
+namespace NitroxPatcher.Patches.Dynamic;
+
+public sealed partial class GoTo_Patch : NitroxPatch, IDynamicPatch
```

Comment:
Patches are named like: `<PatchedClass>_<PatchedMethod>_Patch.cs`. Could you rename the file and class in this fashion?